### PR TITLE
Feature/issue4 partitions presentation

### PR DESCRIPTION
**Pull request by Pergamene**

### CODE REVIEW

File: internal_comps/sp_page/src/components/partitions/sp-inner-partition.js [hunk 1]
```diff
@@ -30,8 +50,30 @@ export class SpInnerPartition extends LitElement {
...
+    }
+  }
+
+  _replaceNewlineWithBreak(partition) {
+    let lines = partition.value.split('\n');
+    if (lines.length > 1) {
+      lines = lines.filter(line => line !== '');
+      return html`<span class="${partition.type}">${lines.map((line) => html`${line}<br>`)}</span>`;
+    }
+    else {
```
Comment (rhyeen) [inline internal_comps/sp_page/src/components/partitions/sp-inner-partition.js]: no need for the `else` statement

**Review by rhyeen — CHANGES_REQUESTED** [review]
For UI changes, we typically have you paste in screenshots that show off the UX changes.  Github lets you paste a screenshot on your clipboard straight into a comment without even having to drag and drop an image.

**Comment by Pergamene** [comment]
image
image
image
image
image

**Comment by rhyeen** [comment]
What's up with the left margins on the headers?

Let's just change the captions to uppercase, the small-caps looks quirky.

**Comment by Pergamene** [comment]
image
They were two different screenshots so I just got more margin in the header image.
image

**Pull request merged**